diff --git a/dist/src/index.js b/dist/src/index.js
index 909d77811704dbd0047680989a137c536d780c04..586db58af91e20eb13690cc2cc45517fff9fbb73 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -97020,6 +97020,13 @@ function handleAttribute(str, attr, parent, preserveCase, svelte5Plus, element)
                 }
                 value.push('})');
             }
+            else if (attr.name.startsWith('mochi:')) {
+                name.unshift('...__sveltets_2_empty({');
+                if (!value) {
+                    value = ['__sveltets_2_any()'];
+                }
+                value.push('})');
+            }
             element.addProp(name, value);
         };
     const transformAttributeCase = (name) => {
